sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDbStateStoreSuite.scala

```diff
@@ -0,0 +1,615 @@
+/*
```

Comment:
This looks like basically copy and paste of StateStoreSuite with some configuration change to enable RocksDbStateStore. Though there're some tests which only make sense to HDFS state store, we should try to deduplicate these things. One good start point of deduplication is, once state store instance is given, we expect it would work similar regardless of provider.

Reply:
Sure. Make sense. I am planning to fix other fixes (and provide performance numbers) before working on deduplicating the test-suite. Let me know if I should fix this first before working on anything else.